scripts/process-stream.sh

```
@@ -329,20 +329,160 @@ echo "## Frames Extracted" >> "$SIGNALS_FILE"
+        local signals=""
+        [ "$vol_spike" = true ] && signals="${signals}VOL "
+        [ "$chat_spike" = true ] && signals="${signals}CHAT "
+        log "  [$SCORE/10 $TYPE ${signals}] $(echo "$header" | sed 's/## //' | cut -c1-40)"
+
+        if [ "${SCORE:-0}" -ge 7 ] 2>/dev/null; then
+            GEM_COUNT=$((GEM_COUNT + 1))
+            echo "### $header $TITLE" >> "$GEMS_FILE"
+            echo "**Score:** $SCORE/10 | **Type:** $TYPE" >> "$GEMS_FILE"
+            [ "$vol_spike" = true ] && echo "**Volume spike:** yes" >> "$GEMS_FILE"
+            [ "$chat_spike" = true ] && echo "**Chat spike:** yes" >> "$GEMS_FILE"
+            echo "" >> "$GEMS_FILE"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)